src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
+          <!-- List Headers -->
+          <v-row class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
```

Comment:
I see you're adding right padding to your columns.

Try removing the "no-gutters" directive and see if that automatically adds in the desired spacing.

Reply:
The design specifically said it wants a minimum of 20px between columns so I thought padding was a more effective way to get that

Reply:
I just noticed this comment is attached to the wrong line number (should be one of the lines with v-col with "pr-5".

I still think you can get the right spacing with the built-in gutters.

Anyway, this is not a deal-breaker. LMK if gutters don't work and we'll leave this as is.